test/settings.py

```diff
@@ -1,11 +1,28 @@
 """Constants used in test files."""
 
-HOST = "127.0.0.1"
-PORT = "5050"
-L1_HOST = "localhost"
-L1_PORT = "8545"
-
-APP_URL = f"http://{HOST}:{PORT}"
-GATEWAY_URL = APP_URL
-FEEDER_GATEWAY_URL = APP_URL
-L1_URL = f"http://{L1_HOST}:{L1_PORT}/"
+import socket
+
+# pylint: disable=invalid-name,too-few-public-methods
```

Comment:
Why is invalid-name check suppressed? Perhaps it can really be a better name, like NetworkSettings.